core/variant.cpp

```diff
@@ -1574,6 +1574,9 @@ Variant::operator String() const {
 					};
 				};
 #endif
+				if (_get_obj().obj->has_method("_to_string")) {
+					return *reinterpret_cast<const String *>(_get_obj().obj->call("_to_string")._data._mem);
```

Comment:
Isn't this dangerous if the script function return something that's not a String?

Reply:
So far it just doesn't print anything, but I could make it test that case, and skip it if it's not a string?

Reply:
So far it just doesn't print anything, but I could make it test that case, and skip it if it's not a string?

Reply:
@vnen since the virtual method on the Object says it should return a string, for me it's returning an empty string, if no string is returned.

I do imagine it is probably a better idea to double check, and I'll add that later today after I get home.

Reply:
Please test with a release build as well.